tooling/nargo_cli/tests/stdlib-tests.rs

```diff
@@ -76,7 +76,7 @@ fn run_stdlib_tests() {
                 &bn254_blackbox_solver::Bn254BlackBoxSolver,
                 &mut context,
                 &test_function,
-                false,
+                true,
```

Comment:
Is this just for local dev?

Reply:
I used it to see `println`. Doesn't seem to have any negative effect when the tests pass.